docker/Dockerfile

```diff
@@ -65,6 +65,8 @@ FROM rosdep-depend AS rosdep-universe-planning-control-depend
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 ARG ROS_DISTRO
 
+COPY src/universe/autoware.universe/launch/tier4_control_launch /autoware/src/universe/autoware.universe/launch/tier4_control_launch
+COPY src/universe/autoware.universe/launch/tier4_planning_launch /autoware/src/universe/autoware.universe/launch/tier4_planning_launch
```

Comment:
tier4_control_launch depends on the following packages from other folders:
* autoware.universe/vehicle/autoware_external_cmd_converter 
* autoware.universe/vehicle/autoware_raw_vehicle_cmd_converter

tier4_planning_launch depends on the following packages from other folders:
* autoware.universe/evaluator/autoware_planning_evaluator 

Should we copy them as well?

Reply:
https://github.com/autowarefoundation/autoware/pull/5586#discussion_r1893364563

Reply:
https://github.com/autowarefoundation/autoware/actions/runs/12413592198/job/34655831530?pr=5587#step:5:1722
```
#55 3.305 ERROR: no rosdep rule for 'autoware_external_cmd_converter'
#55 3.305 ERROR: no rosdep rule for 'autoware_control_evaluator'
#55 3.305 ERROR: no rosdep rule for 'autoware_planning_evaluator'
```